packages/arista_ngfw/manifest.yml

```
@@ -10,7 +10,7 @@ categories:
   - network
 conditions:
   kibana:
-    version: "^8.6.2"
+    version: "^8.10.0"
```

```suggestion
    version: "^8.10.1"
```
This version was [withdrawn](https://www.elastic.co/guide/en/kibana/current/release-notes-8.10.0.html).